english/mom/mom.md

```diff
@@ -0,0 +1,3 @@
+Your mom tells you to stop being a cry baby and go take a walk to clear your head.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.